### PR TITLE
Bump images for the metadata bandaid

### PR DESCRIPTION
**Pull request by BenTheElder**
see: #10796 

/assign @krzyzacy

### CODE REVIEW

**Comment by krzyzacy** [comment]
also set the boolean for verify job?

**Comment by BenTheElder** [comment]
sure, wasn't sure if we w anted that in the same PR or not /shrug

doing that now

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 1]
```diff
@@ -65,6 +33,42 @@ presubmits:
           value: master
         - name: REPO_DIR
           value: /workspace/k8s.io/kubernetes
+        # TODO(bentheelder): remove bandaid once we have this metadata exposed
+        # properly, first-calss
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: class :-)

**Comment by BenTheElder** [comment]
/hold cancel
FYI to @ixdy as oncall

**Comment by ixdy** [comment]
I think you forgot to build a new bootstrap image and make kubekins-e2e use it?

**Comment by ixdy** [comment]
https://github.com/kubernetes/test-infra/blob/7fa56d9358af69640b36e2ee965ed3a170c319da/images/kubekins-e2e/Dockerfile#L18

**Comment by krzyzacy** [comment]
:man_facepalming:

**Comment by BenTheElder** [comment]
:groan:

**Comment by BenTheElder** [comment]
working on it

**Pull request merged**